examples/mobile/HomeApp/webclip/weather/js/app.js

```diff
@@ -14,9 +14,21 @@
 		head.appendChild(jsScript);
 	}
 
+	function changeTheme(event) {
+		var weatherWidget = document.querySelector(".weatherwidget-io");
+
+		if (event.detail.theme === "light") {
+			weatherWidget.setAttribute("data-theme", "pure");
```

Comment:
"pure"?
Do you mean white color?

Reply:
Yes, please find below a list of available styles:
![image](https://user-images.githubusercontent.com/2246331/92203614-dd58d600-ee81-11ea-9fc4-3a03fd4d3e2c.png)

More info here:
https://weatherwidget.io/